custom/enikshay/reports/filters.py

```diff
@@ -0,0 +1,34 @@
+from django.core.urlresolvers import reverse
+
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.reports.filters.base import BaseMultipleOptionFilter
+
+
+class EnikshayLocationFilter(BaseMultipleOptionFilter):
```

Comment:
sorry, probably an obvious question but why can't you use the UCR filter? do you mean the `LocationChoiceProvider`?